packages/vite/rollup.config.ts

```diff
@@ -197,14 +195,10 @@ const cjsConfig = defineConfig({
     publicUtils: path.resolve(__dirname, 'src/node/publicUtils.ts'),
   },
   output: {
-    dir: './dist',
+    ...sharedNodeOptions.output,
     entryFileNames: `node-cjs/[name].cjs`,
     chunkFileNames: 'node-cjs/chunks/dep-[hash].js',
-    exports: 'named',
     format: 'cjs',
-    externalLiveBindings: false,
-    freeze: false,
-    sourcemap: false,
```

Comment:
This is not set in `sharedNodeOptions.output`, but removed because [the default value](https://rollupjs.org/configuration-options/#output-sourcemap) is `false`.